cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -0,0 +1,1148 @@
+						},
+						"key-name-6": {
+							BWItem:     "item-name-3",
+							Attachment: "attachment-name-2",
+						},
+					},
+					To: []SecretContext{
+						{
+							Cluster:   "default",
+							Namespace: "namespace-1",
+							Name:      "prod-secret-1",
+						},
+						{
+							Cluster:   "build01",
+							Namespace: "namespace-2",
+							Name:      "prod-secret-2",
+						},
+					},
+				},
+			},
+			bwClient: bitwarden.NewFakeClient(
+				[]bitwarden.Item{
+					{
+						ID:   "1",
+						Name: "item-name-1",
+						Fields: []bitwarden.Field{
+							{
+								Name:  "field-name-1",
+								Value: "value1",
+							},
+							{
+								Name:  "field-name-2",
+								Value: "value2",
+							},
+						},
+						Attachments: []bitwarden.Attachments{
+							{
+								ID:       "a-id-1-1",
+								FileName: "attachment-name-1",
+							},
+							{
+								ID:       "a-id-1-2",
+								FileName: "attachment-name-2",
+							},
+						},
+					},
+					{
+						ID:   "2",
+						Name: "item-name-2",
+						Fields: []bitwarden.Field{
+							{
+								Name:  "field-name-1",
+								Value: "value3",
+							},
+							{
+								Name:  "field-name-2",
+								Value: "value2",
+							},
+						},
+						Attachments: []bitwarden.Attachments{
+							{
+								ID:       "a-id-2-2",
+								FileName: "attachment-name-2",
+							},
+						},
+					},
+					{
+						ID:   "3",
+						Name: "item-name-3",
+						Fields: []bitwarden.Field{
+							{
+								Name:  "field-name-1",
+								Value: "value1",
+							},
+						},
+						Attachments: []bitwarden.Attachments{
+							{
+								ID:       "a-id-3-1",
+								FileName: "attachment-name-1",
+							},
+							{
+								ID:       "a-id-3-2",
+								FileName: "attachment-name-2",
+							},
+						},
+					},
+				},
+				map[string]string{
+					"a-id-1-1": "attachment-name-1-1-value",
+					"a-id-3-2": "attachment-name-3-2-value",
+				},
+			),
+			expected: fmt.Errorf("not found attachment 'attachment-name-1' in item 'item-name-2'"),
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			fkcDefault := fake.NewSimpleClientset(tc.existSecretsOnDefault...)
+			fkcBuild01 := fake.NewSimpleClientset(tc.existSecretsOnBuild01...)
+			clients := map[string]*fake.Clientset{
+				"default": fkcDefault,
+				"build01": fkcBuild01,
+			}
+			actual := run(nil, tc.config, tc.bwClient, tc.dryRun, getGetFakeSecretGetter(clients))
+			if tc.expected == nil {
+				assert.Nil(t, actual)
+			} else {
+				assert.EqualError(t, actual, tc.expected.Error())
+			}
+			actualSecretsOnDefault, err := fkcDefault.CoreV1().Secrets("").List(metav1.ListOptions{})
+			assert.Nil(t, err)
+			assert.Equal(t, tc.expectedSecretsOnDefault, actualSecretsOnDefault.Items)
+
+			actualSecretsOnBuild01, err := fkcBuild01.CoreV1().Secrets("").List(metav1.ListOptions{})
+			assert.Nil(t, err)
+			assert.Equal(t, tc.expectedSecretsOnBuild01, actualSecretsOnBuild01.Items)
```

Comment:
Use `diff.ObjectReflectDiff` and `reflect.DeepEqual` like we do in other tests. There's no utility to using new dependencies here.